### PR TITLE
Support ppc64le arch compilation and installation

### PR DESCRIPTION
**Pull request by DBres4Power**
## Summary
add "idna==2.10"  to service/service-requirements.txt

## Solution Description
Met such error during 'sh rpm/build.sh rpm_obd': 
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. 
This behaviour is the source of the following dependency conflicts.
requests 2.25.1 requires idna<3,>=2.5, but you have idna 3.6 which is incompatible.

### CODE REVIEW

File: service/service-requirements.txt [hunk 1]
```diff
@@ -6,4 +6,5 @@ uvicorn==0.20.0
 asgi-correlation-id==3.2.1
 starlette_prometheus==0.9.0
 singleton-decorator==1.0.0
-fastapi_utils==0.2.1
\ No newline at end of file
+fastapi_utils==0.2.1
+idna==2.10
```
Comment (frf12) [inline service/service-requirements.txt:10]: fastapi_utils automatically includes idna as a dependency during installation. If you need to specify a particular version of idna, it should be done prior to installing fastapi_utils.

File: service/service-requirements.txt [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
 fastapi==0.87.0
 uvicorn==0.20.0
 asgi-correlation-id==3.2.1
 starlette_prometheus==0.9.0
 singleton-decorator==1.0.0
-fastapi_utils==0.2.1
\ No newline at end of file
+fastapi_utils==0.2.1
+
```
Comment (LINxiansheng) [inline service/service-requirements.txt:11]: delete the empty line

**Comment by LINxiansheng** [comment]
I found you meet the conflict between cython and pyyaml.
Could you add a pre-change logic in rpm/build.sh. When we build obdeploy on ppc64 arch, the build.sh can instead the pyyaml from 6.0.0 to 5.4.1

**Review by LINxiansheng — APPROVED** [review]
LGTM

**Comment by LINxiansheng** [comment]
Thanks for your contribution.

**Pull request merged**